db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/Scheduler.java

```diff
@@ -356,12 +356,12 @@ private void executePickedExecution(Execution execution) {
                 statsRegistry.register(StatsRegistry.ExecutionStatsEvent.COMPLETED);
 
             } catch (RuntimeException unhandledException) {
-                LOG.error("Unhandled exception during execution of task with name '{}'. Treating as failure.", task.get().getName(), unhandledException);
+                LOG.debug("Unhandled exception during execution of task with name '{}'. Treating as failure.", task.get().getName(), unhandledException);
```

Comment:
I can imagine also changing the log level in the default `FailureHandler` implementations (i.e. `OnFailureRetryLater` and `OnFailureReschedule`) to `ERROR` to keep pretty much the same experience for simple tasks using those built-ins, but it's not directly related to the problem this PR aims to solve so I didn't.

Reply:
We discussed this and decided not to go this way.